### PR TITLE
Refactor state render & fix deaddrop accept

### PR DESCRIPTION
**Pull request by Jikstra**
Fixes https://github.com/deltachat/deltachat-desktop/issues/332

### CODE REVIEW

**Comment by ralphtheninja** [comment]
@Jikstra This superseedes #339 I guess?

File: src/renderer/components/SplittedChatListAndView.js [hunk 1]
```diff
@@ -130,6 +130,8 @@ class SplittedChatListAndView extends React.Component {
       showArchivedChats={showArchivedChats}
     />
 
+    console.log(deltachat)
```
Comment (ralphtheninja) [inline src/renderer/components/SplittedChatListAndView.js]: console.log

**Comment by ralphtheninja** [comment]
Too much for me to parse right now.

**Comment by Jikstra** [comment]
> @Jikstra This superseedes #339 I guess?

yes

**Comment by ralphtheninja** [comment]
I'll review this tomorrow with a fresh head.

**Comment by Jikstra** [comment]
> I'll review this tomorrow with a fresh head.

Don't know how/when i will be around today, feel free to merge it/refactor more.

**Comment by r10s** [comment]
GREAT!!1111ELEVEN!ONE!!

**Pull request merged**